metricbeat/tests/system/test_kafka.py

```diff
@@ -46,6 +46,83 @@ def test_partition(self):
 
         self.assert_fields_are_documented(evt)
 
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_consumer(self):
```

Comment:
If these three tests are equal consider using `parameterized`, take a look to `test_ceph.py` or `test_couchbase.py` for  examples.

If you need multiple parameters in principle you can also do it, e.g:
```
@parameterized.expand([
  ('consumer', '8774/tcp'),
  ('producer', '8775/tcp'),
  ('broker', '8779/tcp'),
])
def test_kafka(self, metricset, port):
    ...
```